pkg/controllers/sgx/controller.go

```diff
@@ -36,7 +36,6 @@ import (
 
 const (
 	ownerKey = ".metadata.controller.sgx"
-	amd64    = "amd64"
 )
```

Comment:
the same thing here, please use one line instead of 3

Reply:
Ok, You don't need to comment about this any more.! I am aware of it.!

Reply:
I thought you've missed it as you've updated this PR, but didn't touch this code.

Reply:
Yes, it was another one.. I commented [there](https://github.com/intel/intel-device-plugins-for-kubernetes/pull/764#discussion_r757448723), but I think you couldn't see since I pressed resolve.!
Let me change everything now.!